text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,497 @@
+  channel, as well as from the orderer endpoints. Care must be taken to ensure that the remaining OSNs can still 
+  function and reach consensus.
+- After removal from the consenters set the target OSN switches to "follower" mode and continues to pull blocks from
+  the other orderers.
+- The channel participation admin of the target OSN invokes the "Delete" command with target channel name.
+- If the options parameter indicates not to remove the channel storage, the orderer will not delete 
+the ledger folder of the channel; otherwise, it will delete it. 
+
+### Transitioning to local channel participation management from system-channel-based channel creation
+- Switch the system channel to maintenance mode, in order to stop channel creation TXs from coming in.
+- Configure all OSNs to accept channel participation commands from their respective local orderer channel participation 
+  admins. This is done by changing the `orderer.yaml` and rebooting the OSN. This operation can be staggered, such 
+  that, depending on the fault tolerance setup of respective channels, no channel down-time is experienced.  
+- This will allow OSNs to accept "List" commands and "Delete" of the system channel only. 
+- Delete the system channel from all OSNs.
+- OSNs will now accept the full set of channel participation commands.
+- It is the responsibility of all local orderer channel participation admins to delete the system channel from all OSNs.
+
+### Creating a system channel using the channel participation API
+- Start one or more OSNs without any channels. 
+- These OSNs should be configured to accept channel participation commands from their respective local orderer channel 
+  participation admins. This is achieved by setting the respective fields in the orderer.yaml file of each OSN.
+- These OSNs may belong to different organizations.
+- Generate (using configtxgen, as usual), the genesis block of a system channel. (The system channel contains the 
+  `Consortiums` config group, whereas application channels do not).
+- Create the system channel by invoking the "Join" command as described above, but supply the genesis block of the 
+  system channel, rather than that of an application channel.
+- The channel participation command "Join" will no longer accept commands, and channel creation can only 
+  be done using a config transaction on the system channel.
+- The channel participation commands "List" and "Delete" (of system channel only) will continue to function.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+## The API layer
+We intend to extend the "operations API" (package `core/operations`) by adding a handler for channel participation 
+commands, for path `/channels` as described above. These commands will only be enabled if specified explicitly in the `orderer.yaml` 
+configuration file. These commands will only be available in the orderer and not in the peer.
+
```

Comment:
There is no "operations API". There's an embedded server that this can hook into to provide the orderer API. We must keep them separate.

The implementation also does *not* belong in core/operations.

Reply:
Only the part that accepts the commands is in core/...,  the rest will be under orderer/...